android/build.gradle

```diff
@@ -2,14 +2,14 @@ group 'com.linecorp.flutter_line_sdk'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.7.21'
```

Comment:
Can we keep it at a lower version, such as `1.7.21`? Setting a higher version in the SDK might unintentionally force client apps (e.g., example apps) to use a higher version of dependencies (ex: `kotlin-stdlib-jdk8`).